### ISSUE TITLE
Repair method causes a crash

### ISSUE THREAD

**Issue by xsolutions-taylor** [extremeheat/node-leveldb-zlib#5]
Hi, thanks for these bindings. 

I am trying to use the repair function, but it does not appear to go very far before I get:

**Exception has occurred: Error: napi_get_value_string_utf8(env, argv[0], NULL, 0, &location_size) failed!**

leveldb.js:177 , under async repair(location) . I'd send a pull request myself but I'm not quite sure where to start (if it is indeed a bug).

The method looks static-ish, but I've tried calling it on an instance pointing to a db and also just a plain new instance of LevelDB class:

```
await new LevelDB(db_location).repair(db_location);


and 


await new LevelDB().repair(db_location);
```

Got any clues? Thanks

**Comment by extremeheat** [comment]
Hi, thanks for the report. Can you see if #6 fixes?

**Comment by extremeheat** [comment]
Now released as version 1.1. Please update and reopen this if you still have issues.

**Comment by TaylorDale** [comment]
Thanks for the speedy response! Working perfectly.
